### PR TITLE
Add copyable Nova text field

### LINKED ISSUE #91: Make copyable field open link in new tab
As shown in https://github.com/drewroberts/blog/issues/87:

Screen Shot 2021-04-21 at 11 11 46 PM

The copyable field should also display the url as a clickable hyperlink that opens in a new tab. While `sixlive/nova-text-copy-field` has been valuable in serving as a prebuilt component, it does not support the Nova Text field native method of `asHtml()`, which would normally make it possible to pass in HTML to trigger the hyperlinking of field content.

Started under a custom fork under https://github.com/chx2/nova-text-copy-field, I will need to implement a working `asHTML()` method for any packages that will intend to use the copyable text field.

### PR DESCRIPTION
**Pull request by chx2**
Based on discussion in https://github.com/tipoff/addresses/pull/145, the work I started https://github.com/drewroberts/blog/issues/91 will now be imported into this PR so the files are integrated into the package and do not require sourcing from a custom repo elsewhere.

Closes #91

### CODE REVIEW

File: .github/workflows/php-cs-fixer.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
                 uses: actions/checkout@v1
 
             -   name: Fix style
-                uses: docker://oskarstark/php-cs-fixer-ga
+                uses: docker://oskarstark/php-cs-fixer-ga:2.18.6
```
Comment (chx2) [inline .github/workflows/php-cs-fixer.yml:14]: https://github.com/OskarStark/php-cs-fixer-ga/issues/36

Not really sure what the progress on that will be. This seems to break every other PR as of today...

Comment (joshtorres) [reply .github/workflows/php-cs-fixer.yml:14]: @wolfrednicolas is going through and fixing those for us.

**Review by joshtorres — CHANGES_REQUESTED** [review]
Great work! This is looking good. Would you be able to add some screenshots or a short video of it working in action in this PR? Thank you!

**Comment by chx2** [comment]
Close, but no bullseye.

The field does work natively inside the package so boom, no outdated dependency!

Screen Shot 2021-05-05 at 3 16 47 PM

But, It should be possible to pass in html but this appears to still be escaping it. I will need to do a bit more tweaking to make sure HTML can be passed in.

@joshtorres  Looking foward to the completion of this, I need to use this field inside of Markets, Locations, and any other models with a permalink, should I just use this elswhere or open PR's to add the field in the other packages?

**Comment by joshtorres** [comment]
@phuclh do you have any insight here as we've used a similar copy field many times

**Review by drewroberts — APPROVED** [review]
Thank you!

**Pull request merged**